visualization/app/codeCharta/ui/fileExtensionBar/fileExtensionBar.component.scss

```diff
@@ -37,7 +34,15 @@ file-extension-bar-component {
 		height: 25px;
 		text-align: center;
 		background-color: white;
-		overflow: auto;
+		height: 0px;
+		overflow: hidden;
+		transition: all linear 0.2s;
+
+		&.expanded {
+			border-top: 1px solid #cdcdcd;
+			overflow: auto !important;
```

Comment:
Can we add `overflow` and `border-top` to the `.bar-details` instead of having it inside the `&.expanded` CSS class?

Reply:
the border-top is in expanded, since otherwise we have a too strong border to the ribbon bar when not extended; the overflow is here i think because otherwise it would not hide the labels but show them over the ribbon bar